LEGO1/lego/legoomni/src/race/legoracers.cpp

```diff
@@ -257,10 +275,61 @@ MxU32 LegoRaceCar::HandleSkeletonKicks(float p_param1)
 	return TRUE;
 }
 
-// STUB: LEGO1 0x100131f0
+// FUNCTION: LEGO1 0x100131f0
+// FUNCTION: BETA10 0x100cb88a
 void LegoRaceCar::VTable0x70(float p_float)
 {
-	// TODO
+	if (m_userNavFlag != 0 && (m_userState == LEGORACECAR_KICK1 || m_userState == LEGORACECAR_KICK2)) {
```

Comment:
since this is a boolean, a boolean style check is more appropriate: `!m_userNavFlag` (same in other places)